deps-packaging/leech/cfbuild-leech.spec

```
@@ -0,0 +1,64 @@
+%define leech_version 0.1.21
+
+Summary: CFEngine Build Automation -- leech
+Name: cfbuild-leech
+Version: %{version}
+Release: 1
+Source0: leech-%{leech_version}.tar.gz
+License: LGPL
+Group: Other
+Url: https://github.com/larsewi/leech
```

```suggestion
Url: https://github.com/cfengine/leech
```
or Northern.tech/leech?